README.md

```diff
@@ -41,13 +41,65 @@ $runner->addExtension(new \mageekguy\atoum\ruler\extension($script));
 
 ## Use it
 
-Examples :
+The extension adds a `--filter` option to atoum. This line now appears on the atoum help :
+
+```
+--filter: Filters tests to execute. For example 'not("featureA" in tags) and namespace = "foo\bar"'
+```
+
+You can now filter your tests using any [Hoa\Ruler filter](https://github.com/hoaproject/Ruler).
+
+Those variables are available in the filter :
+* method
+* class
+* namespace
+* testedclass
+* testedclassnamespace
+* tags (as an array)
+
+## Examples
+
+Run all tests except those who have the "needsDatabase" tag:
```

Comment:
Should we put all the quoted parts (“needsDatabase”, “testMethod1”, “testClass1” etc. into `<code />` (with the backtick in Markdown)?